src/features/auth/hooks/useMetamask.ts

```diff
@@ -15,6 +16,11 @@ const useMetamask = () => {
   const accountsChanged = async (newAccount: string) => setAccount(newAccount);
 
   const connect = async () => {
+    if (!navigator.userAgent.toLowerCase().match(/chrome|firefox|edge|brave/)) {
+      dispatch(openToast({ text: 'Navigator is not compatible with MetaMask', type: 'error' }));
```

Comment:
In my opinion, 'Browser' is a more standard way to name web browsers/navigators

Reply:
This change were applied